src/components/microlearning/styles/VideoDetailPage.scss

```diff
@@ -81,4 +81,32 @@
   .vjs-transcribe-cueline {
     color: #00688d;
   }
+  .logo-cutom-style {
```

Comment:
Please use bootstrap utility classes instead of custom css.

Reply:
As the styles are custom so we don't have any specific class for this style.